src/test/demo-sample.spec.js

```diff
@@ -1,15 +1,19 @@
 import { Selector } from "testcafe";
-import { validateIframeTexts, validateIssuerTexts } from "./helper";
+import { validateIframeTexts, validateIssuerTexts, navigateToVerify, location } from "./helper";
 
-fixture("Demo Sample Certificate Rendering").page`http://localhost:3000`;
+fixture("Demo Sample Certificate Rendering").page`${location}/`;
 
 test("demo sample document is rendered correctly when dragged", async (t) => {
+  await navigateToVerify();
   await t.dragToElement(Selector("[draggable='true']"), Selector("#certificate-dropzone"));
   await validateIssuerTexts(["EXAMPLE.OPENATTESTATION.COM"]);
   await validateIframeTexts(["Name & Address of Shipping Agent/Freight Forwarder"]);
 });
 
-test("demo sample document is rendered correctly when clicked", async (t) => {
+// TODO, implement
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("demo sample document is rendered correctly when clicked", async (t) => {
+  await navigateToVerify();
   await t.resizeWindow(800, 600);
   await t.click(Selector("#demoClick"));
```

Comment:
redesign no longer has this mobile button yeah? can remove this test i guess?

Reply:
acutally it does, i just havent added it in yet
![Screenshot 2021-06-23 at 9 37 26 AM](https://user-images.githubusercontent.com/21698179/123021443-b5356600-d406-11eb-9370-2f187e1d6119.png)